web/src/main/java/org/cbioportal/web/SampleController.java

```diff
@@ -100,7 +100,7 @@ private boolean usingAuth() {
             studyIds = studyService
                 .getAllStudies(
                     null,
-                    projection.name(),
+                    Projection.SUMMARY.name(), // force to summary so that post filter doesn't NPE
```

Comment:
I'm having trouble testing this. I would like to write a test where I set the `authenticate` variable (line 60) to a value and then verify that the sampleService is called with the correct projection when I when I do a request with projection META.

I couldn't do this because I didn't see a way to expose the controller so that I could change that field via reflection.